uat/testing-features/src/main/java/com/aws/greengrass/steps/CloudWatchSteps.java

```diff
@@ -86,13 +111,22 @@ public void verifyCloudWatchGroupWithStreams(String componentType, String compon
     }
 
     private boolean doesStreamExistInGroup(String logGroupName, String streamName) {
-        List<LogStream> streams = logsLifecycle.findStream(logGroupName, streamName);
-        boolean exists = streams.stream().anyMatch(stream -> stream.logStreamName().matches(streamName));
+        try {
```

Comment:
This was actually wrong. When the stream was not found immediately it would throw a `ResourceNotFoundException` and immediately stop the tests